lib/twilio/device.ts

```
@@ -797,6 +817,18 @@ class Device extends EventEmitter {
     return payload;
   }
 
+  /**
+   * Destroy the connection to the signaling server.
+   */
+  private _destroyStream() {
+    if (!this.stream) {
+      throw new InvalidStateError('Attempt to destroy non-existent stream.');
```

Why are we throwing here instead of silently ignoring?

---

No particular reason, will change ASAP.